src/Schema/Blueprint.php

```diff
@@ -339,6 +339,16 @@ public function vectorSearchIndex(array $definition, string $name = 'default'):
         return $this;
     }
 
+    /**
+     * Drop an Atlas Search or Vector Search index
+     */
+    public function dropSearchIndex(string $name = 'default'): static
```

Comment:
I think it's sensible to require an explicit value for `$name` since this is a destructive operation. It'd be consistent with PHPLIB's existing API.

Reply:
I assumed that if you don't need to specify the name to create or use the default index, it must be the same for deletion. But as a precaution, it can be required. It's now modified.

Reply:
Requiring explicit values for potentially destructive operations is the same reason the CRUD spec mandates a `filter` parameter for deletes. More generally, this logic can apply to any write operation (we also wouldn't want to default to an empty filter for an `updateMany()`).